Community/Service Requests/service_request_host/js/service_request_host_page.js

```diff
@@ -0,0 +1,70 @@
+// Copyright 2018 BlueCat Networks (USA) Inc. and its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// By: BlueCat Networks
+// Date: 04-05-18
+// Gateway Version: 18.6.1
+// Description: Example Gateway workflows
+
+// JavaScript for your page goes in here.
+
+function updateDeployStatus(deploy_token) {
```

Comment:
Hey I don't understand what this deployment status is for here. It doesn't seem to be part of the main logic of the workflow at all. Can you clarify why this and the server side endpoint are part of this workflow? Thanks.

Reply:
That was probably left over from a previous workflow I based this off of and I just missed it. I will remove this and recommit it.

Reply:
Hi Chris, looks like updateDeployStatus is still there.

Reply:
My apologies Chris, I looked at these files and not your very latest commits.